content/english/updates/Publications_update.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Accessing more Swedish research"
+date: 2025-10-30
+summary: The portal is currently undergoing a transition to be able to show more Swedish data and research.
+---
+
+The Swedish Pathogens Portal aims to make outputs from Swedish research on infectious disease and pandemic preparedness as Open and FAIR as possible. Multiple major developments will take place in the next few months to maximise the levels of data and tools that can be accessed directly through the portal.
+
+As an initial step, and in line with ongoing efforts to diversify the content of the portal, the [publications section](/publications) has been updated to show publications on multiple different pathogens and topics (e.g. influenza and antibiotic resistance). The page shows the 10 most recent Swedish publications on a given pathogen/topic from [Europe PMC](https://europepmc.org). A direct link to more publications on a given pathogen/topic is available on the page.
+
+The publications section previously focused on COVID-19 and SARS-CoV-2. The publications were categorised according to the research field (e.g. social science, health), publication type, and funder. This data can still be accessed via our [publications database](https://publications-covid19.scilifelab.se/), and is presented on our [publications dashboard](http://localhost:1313/dashboards/covid_publications/).
```

Comment:
we should not have `localhost` here (probably unintended), just `/dashboards/covid_publications/` should suffice.